spec/index.bs

```diff
@@ -980,6 +980,34 @@ The default is used if a value is not explicitly specified in {{GPUDeviceDescrip
         <tr><th>Limit name <th>Type <th>[=limit/Better=] <th>[=limit/Default=]
     </thead>
 
+    <tr><td><dfn>maxTextureDimension1D</dfn>
+        <td>{{GPUSize32}} <td>Higher <td>4096
+    <tr class=row-continuation><td colspan=4>
+        The maximum allowed {{GPUTextureDescriptor/size}}.{{GPUExtent3DDict/width}}
```

Comment:
this is technically incorrect, since the `size` has type `GPUExtent3D`, not `GPUExtent3DDict`

Reply:
It looks like that `GPUExtent3D` is typedef-ed as `GPUExtent3DDict`.

Reply:
It's defined as:
```
typedef (sequence<GPUIntegerCoordinate> or GPUExtent3DDict) GPUExtent3D;
```

There are helper defines though, but to use them your text would need to be changed to this:
```
{{GPUTextureDescriptor/size}}.[=Extent3D/width=]
```

Reply:
Done. It works!